platform/broadcom/sai-xgs.mk

```diff
@@ -1,27 +1,27 @@
+# Broadcom XGS SAI definitions
 LIBSAIBCM_XGS_VERSION = 14.1.0.1.0.0.0.1
-LIBSAIBCM_DNX_VERSION = 14.1.0.1.0.0.0.0
 LIBSAIBCM_XGS_BRANCH_NAME = SAI_14.1.0_GA
-LIBSAIBCM_DNX_BRANCH_NAME = SAI_14.1.0_GA
 
 LIBSAIBCM_XGS_URL_PREFIX = "https://packages.trafficmanager.net/public/sai/sai-broadcom/$(LIBSAIBCM_XGS_BRANCH_NAME)/$(LIBSAIBCM_XGS_VERSION)/xgs"
-LIBSAIBCM_DNX_URL_PREFIX = "https://packages.trafficmanager.net/public/sai/sai-broadcom/$(LIBSAIBCM_DNX_BRANCH_NAME)/$(LIBSAIBCM_DNX_VERSION)/dnx"
 
+# Runtime package
 BRCM_XGS_SAI = libsaibcm_$(LIBSAIBCM_XGS_VERSION)_amd64.deb
 $(BRCM_XGS_SAI)_URL = "$(LIBSAIBCM_XGS_URL_PREFIX)/$(BRCM_XGS_SAI)"
+
+# Development package
 BRCM_XGS_SAI_DEV = libsaibcm-dev_$(LIBSAIBCM_XGS_VERSION)_amd64.deb
 $(eval $(call add_derived_package,$(BRCM_XGS_SAI),$(BRCM_XGS_SAI_DEV)))
 $(BRCM_XGS_SAI_DEV)_URL = "$(LIBSAIBCM_XGS_URL_PREFIX)/$(BRCM_XGS_SAI_DEV)"
 
-# SAI module for DNX Asic family
-BRCM_DNX_SAI = libsaibcm_dnx_$(LIBSAIBCM_DNX_VERSION)_amd64.deb
-$(BRCM_DNX_SAI)_URL = "$(LIBSAIBCM_DNX_URL_PREFIX)/$(BRCM_DNX_SAI)"
-
+# Package registration
 SONIC_ONLINE_DEBS += $(BRCM_XGS_SAI)
-SONIC_ONLINE_DEBS += $(BRCM_DNX_SAI)
+
+# Dependencies
 $(BRCM_XGS_SAI_DEV)_DEPENDS += $(BRCM_XGS_SAI)
 
-$(BRCM_XGS_SAI)_SKIP_VERSION=y
-$(BRCM_XGS_SAI_DEV)_SKIP_VERSION=y
-$(BRCM_DNX_SAI)_SKIP_VERSION=y
+# Version handling
+$(BRCM_XGS_SAI)_SKIP_VERSION = y
```

Comment:
Better to follow previous format, if want to make it more readable, please make sure the space here works fine,  "$(BRCM_XGS_SAI)_SKIP_VERSION = y"

Reply:
good catch, removed the space to follow previous format.